README.md

```diff
@@ -38,32 +38,39 @@ In order to use this library you can either use Mixins or have your models inher
    from django_multitenant.fields import *
    from django_multitenant.models import *
    ```
-1. All models should inherit the TenantModel class.
+2. All models should inherit the TenantModel class.
    `Ex: class Product(TenantModel):`
-1. Define a static variable named tenant_id and specify the tenant column using this variable.
-   `Ex: tenant_id='store_id'`
-1. All foreign keys to TenantModel subclasses should use TenantForeignKey in place of
+3. Define a static variable named tenant_id and specify the tenant column using this variable.You can define tenant_id in three ways. Any of them is acceptavle 
+   * Using TenantMeta.tenant_field_name variable
+   * Using TenantMeta.tenant_id variable
+   * Using template_id field
+> **Warning**
```

Comment:
This warning should be indented so that it appears as a child of the list item.

Also, it says "template_id" did you mean "tenant_id" ?

Reply:
Fixed